### ISSUE TITLE
Elements must have sufficient color contrast for usermanagement

### ISSUE THREAD

**Issue by bryantal** [BCStudentSoftwareDevTeam/cas#315]
**[Original report](https://bitbucket.org/laborstudents/cas-flask/issue/315) by Jamal Aw Yonis (Bitbucket: [awyoonisj](https://bitbucket.org/awyoonisj), GitHub: [jawyoonis](https://github.com/jawyoonis)).**

----------------------------------------

**Issue description**:

Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds. The color contras should be at least **4:5:1**  rotion for small text and** 3:1** for large text. To learn more about the issue GO TO THIS  [ LINK ](https://dequeuniversity.com/rules/axe/3.1/color-contrast?application=AxeChrome)

Files effected: usermanagement.html/js/css 
![photo.PNG](https://bitbucket.org/repo/y4axrX/images/1132482801-photo.PNG)
